src/styles/_catalog.scss

```diff
@@ -1,9 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, $cell-size);
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $cell-size);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $cell-size);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $cell-size);
+  }
 
   &__card {
     display: flex;
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/195805723-d79b0cf0-4644-4970-81a5-36ac6ee4ebfe.png)
You can also set card's width and height using variables